NEWS

```diff
@@ -1,3 +1,40 @@
+26-Jan-2017 IGNITION v0.21.0
+
+  Features
+
+    - Add support for networkd drop-ins
+    - Add new binary, ignition-validate, for validating Ignition configs
+    - Add overwrite field to files, directories, and links for deleting
```

Comment:
Here and below, can you please add quotes around field names? Without those, parsing these sentences is quite hard for a non-native reader.